crates/nix_rs/src/flake/mod.rs

```diff
@@ -39,7 +39,13 @@ impl Flake {
         // TODO: Can we cache this?
         let nix_config = NixConfig::from_nix(nix_cmd).await?;
         let system = nix_config.system.value;
-        let output = FlakeOutputs::from_nix(nix_cmd, &url).await?;
+        let nix_flake_schemas_cmd = NixCmd {
+            command: "nix-flake-schemas".to_string(),
+            extra_experimental_features: vec!["nix-command".to_string(), "flakes".to_string()],
+            extra_access_tokens: vec![],
+            refresh: false,
+        };
```

Comment:
You can derive this from existing `nix_cmd` (Rust immutable struct update). 

Also, instead of `"nix-flake-schemas".to_string()` we should use the actual nix store path specified at compile time (like `DEVOUR_FLAKE` in `nixci`). That env var can also specify the path to use to invoke it when ran as static binary (#158)?